Dockerfile

```diff
@@ -65,10 +65,11 @@ ENTRYPOINT ["./web-api"]
 FROM scratch AS web-frontend-release
 
 COPY --from=builder /etc/passwd /etc/passwd
-COPY --from=builder --chmod=777 /app/web-frontend web-frontend
-COPY --from=web-frontend-builder /app/out .
+COPY --from=builder --chmod=777 /app/web-frontend /web-frontend
+COPY --from=web-frontend-builder /app/out /out
```

Comment:
changed as https://github.com/ucl-arc-tre/portal/blob/7818e8a154c52e1119206fcf7085b6c747f9965f/cmd/web-frontend/main.go#L21 now loops recursively and there are files which it doesn't have permission to view

Reply:
All still works locally for me 👍